### ISSUE TITLE
dependency on ruaml.yaml 0.15

### ISSUE THREAD

**Issue by omry** [episodeyang/ml_logger#14]
ruaml.yaml states:
> For production systems you should pin the version being used with ``ruamel.yaml<=0.15

By not enforcing a version, you install the latest version which is API incompatible with anything before 0.15.
This means that systems that depends on 0.14 can't work with ml logger (such as my own project, which has a dependency on OmegaConf, which depends on ruaml.yaml 0.14).

Suggested fixes:
1. Force version <= 0.15 and ensure the code works.
2. switch to pyyaml.
3. do a local import from ruaml.yaml, at least this way it would not break on initialization.

stacktrace:
```
$ python mbrl.py  train -e MBRLCartpole-v0 env.instances=2
Traceback (most recent call last):
  File "mbrl.py", line 16, in <module>
    from ml_logger import logger
  File "/nfs/home/omry/.conda/envs/pets/lib/python3.6/site-packages/ml_logger/__init__.py", line 2, in <module>
    from .log_client import LogClient
  File "/nfs/home/omry/.conda/envs/pets/lib/python3.6/site-packages/ml_logger/log_client.py", line 5, in <module>
    from ml_logger.server import LoggingServer
  File "/nfs/home/omry/.conda/envs/pets/lib/python3.6/site-packages/ml_logger/server.py", line 5, in <module>
    from ruamel.yaml import YAML, StringIO
ImportError: cannot import name 'YAML'
```

**Comment by episodeyang** [comment]
Thanks Omry! I will try to get to this tonight.

**Comment by episodeyang** [comment]
@omry is 0.15.0 really fine? I am on 0.15.0 atm and it seems that what you really want is `<0.15.0` instead of `<=`.

**Comment by omry** [comment]
nope, that was a mistake. meant < 0.15

**Comment by episodeyang** [comment]
This is addressed in v0.3.3. Thanks!
